app/coffee/DocumentUpdaterManager.coffee

```diff
@@ -55,10 +55,8 @@ module.exports = DocumentUpdaterManager =
 	queueChange: (project_id, doc_id, change, callback = ()->)->
 		jsonChange = JSON.stringify change
 		doc_key = "#{project_id}:#{doc_id}"
-		multi = rclient.multi()
-		multi.rpush Keys.pendingUpdates({doc_id}), jsonChange
-		multi.sadd  "DocsWithPendingUpdates", doc_key
```

Comment:
No, it's got 4.5 million entries in the doc updater because real-time is adding to it, but doc updater does nothing with it.